### ISSUE TITLE
tensorflow lite become slower after quantization

### ISSUE THREAD

**Issue by JPPolnareff** [tensorflow/tensorflow#35556]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): source
- TensorFlow version (use command below): 1.15 and 2.0
- Python version: using C++
- Bazel version (if compiling from source): 0.26 for tf1.5/ 1.10 for tf2.0
- GCC/Compiler version (if compiling from source): 4.9.4
- CUDA/cuDNN version:
- GPU model and memory:

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**

**Describe the expected behavior**

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

I'm using tensorflow lite for quantizing mobileNetV1 to int8. And then, I do inference using label_image.cc in tensorflow github source code. All the work is done on my Ubuntu16.04, on CPU, not GPU. Before quantization, doing inference takes 85ms. However, the quantized model takes 1200ms. I do the experiments using tf1.15/tf2.0, both of them have such issue. Really wired, quantized model should be faster.

**Comment by ravikyram** [comment]
@JPPolnareff 

Will it be possible to share the related code. It helps us in localizing the issue faster.Thanks!

**Comment by JPPolnareff** [comment]
[label_image.zip](https://github.com/tensorflow/tensorflow/files/4024359/label_image.zip)
@ravikyram 

Thanks for your reply. The attached file contain the .cc file, and two tflite model (unquantized/ quantized respectively). 
The command I use is:
(compile label_image.cc)
bazel build -c opt --cxxopt=-std=c++11 tensorflow/lite/examples/label_image:label_image
(run unquantized model)
bazel-bin/tensorflow/lite/examples/label_image/label_image   --tflite_model /headless/Desktop/keras_pruning/1.0/220.tflite  --labels /tmp/labels.txt   --image tensorflow/lite/examples/label_image/testdata/grace_hopper.bmp --count 10 --threads 1 #--verbose 1 
(run quantized model)
bazel-bin/tensorflow/lite/examples/label_image/label_image   --tflite_model /headless/Desktop/keras_pruning/1.0/220_int8_force.tflite  --labels /tmp/labels.txt   --image tensorflow/lite/examples/label_image/testdata/grace_hopper.bmp --count 10 --threads 1 #--verbose 1 

For installing label_image demo,
the reference for original label_image.cc demo is: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/label_image

**Comment by alanchiao** [comment]
For quantizing the TFLite model, are you using the weight quantization or full integer quantization path? https://www.tensorflow.org/lite/performance/post_training_quantization#optimization_options
